stock_picking_unreserve_line/readme/DESCRIPTION.rst

```diff
@@ -0,0 +1 @@
+Permits to unreserve stock.move
```

Comment:
Can you be a bit more verbose? What can the user expect after the installation of the module?